glib/Idle.cs

```diff
@@ -44,44 +50,84 @@ protected override bool Invoke ()
 				return real_handler ();
 			}
 		}
-		
+
 		private Idle ()
 		{
 		}
-		
-		[DllImport("libglib-2.0-0.dll", CallingConvention=CallingConvention.Cdecl)]
-		static extern uint g_idle_add (SourceProxy.GSourceFuncInternal d, IntPtr data);
+
+		const int G_PRIORITY_DEFAULT_IDLE = 200;
```

Comment:
Should I add a glue method to invoke on static cctor so we get the value of this define?